src/components/PageBody/TrackPageViewConditional.tsx

```diff
@@ -0,0 +1,31 @@
+import { headers } from 'next/headers';
+
+import { SiteContentPointer } from '@/lib/api';
+
+import { TrackPageView } from './TrackPageView';
+
+export function TrackPageViewConditional(props: {
+    apiHost: string;
+    sitePointer?: Pick<SiteContentPointer, 'siteId' | 'organizationId'>;
+    spaceId: string;
+    pageId: string | undefined;
+}) {
+    const { apiHost, sitePointer, spaceId, pageId } = props;
+
+    const headerSet = headers();
+    if (
+        process.env.GITBOOK_BLOCK_PAGE_VIEWS_TRACKING &&
+        !headerSet.has('x-gitbook-track-page-views')
```

Comment:
This makes it on by default. I'm conflicted if that's best.

Reply:
I think it should be on by default 👍  just like SEO indexation